test/unit/document/document_test.ts

```diff
@@ -75,6 +75,41 @@ describe('DocumentReplica', function () {
     assert.equal('{"data":{"":null,"null":null}}', doc.toSortedJSON());
   });
 
+  it('delete elements of array test', function () {
```

Comment:
I think there is a same test :) https://github.com/yorkie-team/yorkie-js-sdk/pull/185/files#diff-da39b353679eb792fb893b2585229d6a036f56136e143739ee724a048658b3baR78-R83

Reply:
When I applied two PRs at the same time, I tried to make sure there were no problems.